src/containers/Header/NavigationMenu/NavigationMenu.scss

```diff
@@ -0,0 +1,228 @@
+@use 'sass:math';
+
+@import '../../shared/css/variables';
+
+$menu-toggle-size: 25px;
+$menu-toggle-line-spacing: math.div($menu-toggle-size, 4);
+$menu-toggle-line-length: $menu-toggle-size - $menu-toggle-line-spacing;
+
+.topbar,
+.navbar {
+  display: flex;
+  width: 100%;
+}
+
+.navbar {
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 0 auto;
+  gap: 0 16px;
+
+  .navbar-nav a {
+    display: block;
+  }
+}
+
+.navbar-brand {
+  display: flex;
+  align-items: center;
+
+  svg {
+    height: 26px;
+    color: #fff;
+  }
+}
+
+.navbar-collapse {
+  display: none;
+  width: 100%;
+  height: 100%;
+  flex-basis: 100%;
+  margin-left: auto;
+}
+
+.nav-search {
+  flex-grow: 1;
+}
+
+.nav-link {
+  color: $white;
+  font-size: 14px;
+}
+
+/* Nav menu */
+.navbar-nav {
+  display: block;
+  padding: 0;
+  margin: 0 auto 0 0;
+  list-style: none;
+
+  .btn {
+    padding: 0;
+    border: none;
+    font-weight: normal;
+  }
+
+  .dropdown-toggle {
+    gap: 8px;
+
+    &:hover {
+      background: transparent;
+    }
+  }
+}
+
+/* Menu Icon */
+.navbar-toggle {
+  display: flex;
+  width: $menu-toggle-size;
+  height: $menu-toggle-size;
+  align-items: center;
+  justify-content: center;
+  margin-left: auto;
+  cursor: pointer;
+}
+
+.navbar-toggle-line {
+  position: relative;
+  display: block;
+  width: $menu-toggle-line-length;
+  height: 2px;
+  background: white;
+
+  &::before,
+  &::after {
+    position: absolute;
+    display: block;
+    width: 100%;
+    height: 100%;
+    background: white;
+    content: '';
+    transition: all 0.2s ease-out;
+  }
+
+  &::before {
+    top: $menu-toggle-line-spacing;
+  }
+
+  &::after {
+    top: -$menu-toggle-line-spacing;
+  }
+}
+
+/* Open Mobile Menu */
+.navbar-toggle-state:checked ~ .navbar-toggle {
+  .navbar-toggle-line {
+    width: 51px; // sqrt(($menu-toggle-line-length)^2 * 2)
+    background: transparent;
+
+    &::before {
+      top: 0;
+      transform: rotate(-45deg);
+    }
+
+    &::after {
+      top: 0;
+      transform: rotate(45deg);
+    }
+  }
+}
+
+// Prevents navigation from keeping styles when menu is opened on mobile sizes and then resized larger
+@media (max-width: $tablet-landscape-upper-boundary) {
+  .navbar-collapse {
+    max-height: 100%;
+    margin-top: 24px;
+
+    .navbar-toggle-state:checked ~ & {
```

Comment:
What does this syntax let you do?

Reply:
When the current scoped element is a sibling to the hidden toggle checkbox that is checked style it a certain way.

This is a technique for pure css menus.